index.html

```diff
@@ -199,8 +199,8 @@ <h3 class="features__title">We like blocks, elements and modifiers.</h3>
     <div class="footer__copyright">
       Leean is under <a rel="nofollow" href="https://www.gnu.org/licenses/gpl-2.0.txt">GPL</a> license.
     </div>
-    <div class="footer__brand--moxie">
-      <img src="src/images/moxie-footer.png" alt="" class="footer_brand__media">
+    <div class="footer__brand--nolte">
+      <img src="src/images/nolte-footer.png" alt="" class="footer_brand__media">
```

Comment:
I did not. They have the moxie logo so I think there would have to be new files.